realm/realm-library/src/syncIntegrationTest/java/io/realm/objectserver/utils/HttpUtils.java

```diff
@@ -60,20 +68,32 @@ public static void startSyncServer() throws Exception {
     // Checking the server
     private static boolean waitAuthServerReady() throws InterruptedException {
         int retryTimes = 50;
+
+        // Dummy invalid request, which will trigger a 400 (BAD REQUEST), but indicate the auth
+        // server is responsive
         Request request = new Request.Builder()
-                .url(Constants.AUTH_SERVER_URL)
+                .post(RequestBody.create(MediaType.parse("application/json; charset=utf-8"), ""))
+                .url(Constants.AUTH_URL)
                 .build();
 
         while (retryTimes != 0) {
+            Response response = null;
             try {
-                Response response = client.newCall(request).execute();
-                if (response.isSuccessful()) {
+                response = client.newCall(request).execute();
+                if (response.code() == 400) {
                     return true;
                 }
                 RealmLog.error("Error response from auth server: %s", response.toString());
             } catch (IOException e) {
-                RealmLog.error(e);
+                // TODO As long as the auth server hasn't started yet, OKHttp cannot parse the response
+                // correctly. At this point it is unknown weather is a bug in OKHttp or how an
+                // unknown host is reported. Disabling the log out put for now as it just pollutes the log.
+                // RealmLog.error(e);
```

Comment:
OkHttp was flooding the log with errors during the setup phase while waiting for the server to come online. So it obscured any real errors happening. I can add it back again though